examples/mag/README.md

```diff
@@ -44,3 +44,142 @@ python3 -m graphstorm.run.gs_link_prediction \
+python3 -m graphstorm.run.gs_node_classification \
+            --num-trainers 8 \
+            --num-servers 4 \
+            --num-samplers 0 \
+            --part-config mag_4parts/mag.json \
+            --ip-config ip_list_4p.txt \
+            --cf mag_gnn_nc.yaml
+```
+
+This method leads to the accuracy of 53.78%.
+
+We can replace RGCN with HGT.
+
+```
+python3 -m graphstorm.run.gs_node_classification \
+			--num-trainers 8 \
+			--num-servers 4 \
+			--num-samplers 0 \
+			--part-config mag_4parts/mag.json \
+			--ip-config ip_list_4p.txt \
+			--cf mag_gnn_nc.yaml \
+			--model-encoder-type hgt
+```
+On the full graph, HGT has much better performance than RGCN. The model reaches the accuracy of 59.12%.
+
+### Fine-tune BERT model to predict the venue
+
+We can fine-tune the BERT model on the paper nodes and predict the venue directly.
+This can be done in GraphStorm with the following command.
+
+```
+python3 -m graphstorm.run.gs_node_classification \
+		--num-trainers 8 \
+		--num-servers 1 \
+		--num-samplers 0 \
+		--part-config mag_4parts/mag.json \
+		--ip-config ip_list_4p.txt \
+		--cf mag_bert_nc.yaml \
+		--save-model-path mag_bert_nc_model
+```
+
+The accuracy is 41.88.
+
+### Fine-tune BERT model on the graph data and train GNN model to predict the venue
+
+To achieve good performance, we should fine-tune the BERT model on the graph data.
+One way of fine-tuning the BERT model on the graph data is to fine-tune the BERT model
+with link prediction. This can be done in GraphStorm with the following command.
+
+```
+python3 -m graphstorm.run.gs_link_prediction \
+			--num-trainers 8 \
+			--num-servers 1 \
+			--num-samplers 0 \
+			--part-config mag_4parts/mag.json \
+			--ip-config ip_list_4p.txt \
+			--cf mag_bert_ft.yaml \
+			--save-model-path mag_bert_lp_model
+```
+
+We can load the fine-tuned BERT model to generate BERT embeddings and train GNN model.
+
+```
+python3 -m graphstorm.run.gs_node_classification \
+            --num-trainers 8 \
+            --num-servers 4 \
+            --num-samplers 0 \
+            --part-config mag_4parts/mag.json \
+            --ip-config ip_list_4p.txt \
+            --cf mag_gnn_nc.yaml \
+            --restore-model-path mag_bert_lp_model/epoch-2 \
+            --restore-model-layers dense_embed
+
+python3 -m graphstorm.run.gs_node_classification \
+            --num-trainers 8 \
+            --num-servers 4 \
+            --num-samplers 0 \
+            --part-config mag_4parts/mag.json \
+            --ip-config ip_list_4p.txt \
+            --cf mag_gnn_nc.yaml \
+            --restore-model-path mag_bert_lp_model/epoch-2 \
+            --restore-model-layers dense_embed \
+			--model-encoder-type hgt
+```
+
+The accuracy of RGCN with the BERT model fine-tuned with link prediction is 57.86%,
+while the accuracy of HGT is 57.86%.
+
+We can also train a GNN model with the BERT model fine-tuned for predicting venues.
+
+```
+python3 -m graphstorm.run.gs_node_classification \
```

Comment:
What is the difference between this one and L132?

Reply:
this one trains GNN model with BERT embeddings fine-tuned with node classification.
L132 uses the BERT embeddings fine-tuned with link prediction.